src/App.tsx

```diff
@@ -15,48 +16,147 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortFields {
+  byAlfabet = 'alphabet',
+  byLength = 'length',
+  byIndexdesc = 'reverse default',
```

Comment:
it shouldn't be here - just reverse array sorted by other properties if it's needed